dockers/tools/cli/loadtest/Dockerfile

```diff
@@ -118,9 +118,9 @@ RUN GO_VERSION="$(< GO_VERSION)" \
     && upx -9 -o "/usr/bin/${APP_NAME}" "${APP_NAME}"
 
 # Start From Scratch For Running Environment
-FROM --platform=${BUILDPLATFORM} ubuntu:devel
+FROM --platform=${TARGETPLATFORM} ubuntu:devel
 # Start From Alpine For Debug Environment
-# FROM --platform=${BUILDPLATFORM} alpine:latest
+# FROM --platform=${TARGETPLATFORM} alpine:latest
```

Comment:
Do we have to leave these comments?

Reply:
Thank you for your suggestion, I removed the unnecessary comment.